app/main.py

```diff
@@ -1,13 +1,13 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
+    # write your code here
     pass
```

Comment:
The `format_linter_error` function is currently empty. You need to implement the logic for this function.